health/health.c

```diff
@@ -140,7 +140,7 @@ void health_reload_host(RRDHOST *host) {
     }
 
     //Reload the section "host labels" from netdata.conf, because it can be new labels that will indicate alarms.
-    config_reload_section(NULL, CONFIG_SECTION_HOST_LABEL);
+    config_load(NULL, 1, CONFIG_SECTION_HOST_LABEL);
```

Comment:
I don't think this call should be here?

Reply:
Case we reload all alarms, they will need to have all the labels updated before we apply them. I decided to set it here to avoid two calls to `netdatacli`, the first would be to reload the labels and after this we would need another to call the alarm reload. I agree  that your comment is more atomic, but should we keep this to simplify users life?

Reply:
Updating the label config is an explicit operation by the user, it should not be a side-effect of reloading the alarms - it makes it harder for the user to understand why something has happened.

Reply:
All right, I removed it from health in the last commit.